src/Aura/MakePkg.hs

```diff
@@ -31,27 +31,26 @@ module Aura.MakePkg
 
 import           BasicPrelude hiding (FilePath)
 
-import qualified Data.Text as T
 import           Shelly
-import           Text.Regex.PCRE ((=~))
-
-import           Aura.Shell
+import           Filesystem.Path.CurrentOS
 
 import           Utilities
+import           Aura.Shell
 
 ---
 
 type User = Text
 
+findPkgFile :: FilePath -> Sh [FilePath]
+findPkgFile = findDirFilterWhen (pure . const False) (pure . matches)
+  where matches fp = ["xz", "tar", "pkg"] `isPrefixOf` reverse (extensions fp)
```

Comment:
I don't think so. Also, I don't think that the old code did that. OTOH, I gooffed and that should be "src" and not "pkg".